README.md

```diff
@@ -62,6 +62,19 @@ Third option, do a az login:
 
 You also could develop your Ansible playbook and run it in [Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=vscoss.vscode-ansible).
 
+### Using Azure cloud hell in Visual Studio Code
+
+Azure cloud shell automatically logs you in your azure subscription so you need not to do anything extra other than login to cluod shell and run the ansible-playbook command with -e switch to provide
+resource_group_name variable value. 
+
+To be able to run Cloud Shell in Visual Studio Code, you will need to install [Azure Account Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.azure-account) in VS Code.
```

Comment:
Here we can use the [Ansible VSCode extension](https://marketplace.visualstudio.com/items?itemName=vscoss.vscode-ansible) to do the auth and upload  file to cloud shell, and this extension will stream the log to vscode console. Besides this, the Ansible extension has support some Ansible specific feature, like auto completion for the playbook. So here we recommend to use the Ansible extension rather than Azure Account Extension.